sklearn/utils/tests/test_random.py

```diff
@@ -180,3 +183,24 @@ def test_random_choice_csc_errors():
     class_probabilities = [np.array([0.5, 0.6]), np.array([0.6, 0.1, 0.3])]
     assert_raises(ValueError, random_choice_csc, 4, classes,
                   class_probabilities, 1)
+
+
+@pytest.mark.parametrize("low,high,base", [
+    (-1, 0, 10),
+    (0, 2, np.exp(1)),
+    (-1, 1, 2),
+])
+def test_log_uniform(low, high, base):
+    rv = loguniform(low, high, base=base)
+    rvs = rv.rvs(size=100)
+    assert_true((base**low <= rvs).all() and (rvs <= base**high).all())
```

Comment:
With the adoption of pytest, we are phasing out use of test helpers `assert_equal`, `assert_true`, etc. Please use bare `assert` statements, e.g. `assert x == y`, `assert not x`, etc.